### PR TITLE
Cranial Remote Deck Cyberware (CRD)

### PR DESCRIPTION
**Pull request by TheVilest**
You down with CRD? Yeah you know me~

### CODE REVIEW

File: src/act.drive.cpp [hunk 1]
```diff
@@ -888,6 +888,11 @@ ACMD(do_control)
         if (GET_OBJ_TYPE(GET_EQ(ch, x)) == ITEM_RCDECK)
           has_rig = TRUE;
 
+  if (!has_rig)
+    for (cyber = ch->cyberware; cyber; cyber = cyber->next_content)
+      if (GET_OBJ_VAL(cyber, 0) == CYB_CRD))
```
Comment (luciensadi) [inline src/act.drive.cpp:893]: `GET_CYBERWARE_TYPE` instead of the raw obj_val call, please!

File: src/act.drive.cpp [hunk 2]
```diff
@@ -955,7 +960,7 @@ ACMD(do_control)
 ACMD(do_subscribe)
 {
   struct veh_data *veh = NULL;
-  struct obj_data *obj;
+  struct obj_data *obj, *cyber;
```
Comment (luciensadi) [inline src/act.drive.cpp:963]: If you're not referencing this value anywhere other than the for loop, you can just initialize it there.
`for (struct obj_data *cyber = ch->cyberware;...`

File: src/db.cpp [hunk 3]
```diff
@@ -6286,6 +6286,22 @@ void price_cyber(struct obj_data *obj)
           break;
       }
       break;
+    case CYB_CRD:
+      GET_CYBERWARE_ESSENCE_COST(obj) = 30;
+      GET_OBJ_AVAILTN(obj) = 4;
+      GET_OBJ_AVAILDAY(obj) = 3;
+      switch (GET_OBJ_VAL(obj, 1)) {
```
Comment (luciensadi) [inline src/db.cpp:6293]: `GET_CYBERWARE_RATING` please

**Review by luciensadi — CHANGES_REQUESTED** [review]
Looks pretty tight! Few nits and this should be good to go.

File: src/act.drive.cpp (hunk 2 above)
Comment (TheVilest) [reply src/act.drive.cpp]: That's pretty useful info going forward

File: src/act.drive.cpp (hunk 1 above)
Comment (TheVilest) [reply src/act.drive.cpp]: Is the `(cyber, 0)` value reference still needed to reference type?

File: src/db.cpp (hunk 3 above)
Comment (TheVilest) [reply src/db.cpp]: Gotcha

File: src/act.drive.cpp (hunk 1 above)
Comment (TheVilest) [reply src/act.drive.cpp]: I'm assuming not, since utils.h has a convenience definition

**Pull request merged**